mail-filter/sqlgrey/sqlgrey-1.8.0.ebuild

```diff
@@ -0,0 +1,246 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils systemd user
+
+DESCRIPTION="SQLgrey is a postfix policy service implementing a grey-listing policy"
+SRC_URI="mirror://sourceforge/sqlgrey/sqlgrey-1.8%20%28stable%29/${P}.tar.gz"
+HOMEPAGE="http://sqlgrey.sourceforge.net/"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="mysql postgres sqlite"
+RDEPEND="dev-lang/perl
+	dev-perl/DBI
+	dev-perl/Net-Server
+	dev-perl/Date-Calc
+	virtual/mailx
+	acct-user/sqlgrey
+	postgres? ( dev-perl/DBD-Pg )
+	sqlite? ( dev-perl/DBD-SQLite )
+	mysql? ( dev-perl/DBD-mysql )
+	!postgres? ( !mysql? ( !sqlite? ( dev-perl/DBD-Pg ) ) )"
+DEPEND="$RDEPEND
+	sys-apps/sed"
+KEYWORDS="~amd64 ~arm ~sparc ~x86"
+DOCS="HOWTO FAQ README README.OPTINOUT README.PERF TODO Changelog"
+PATCHES=(
+	"${FILESDIR}/sqlgrey-1.8.0-init-openrc.patch"
+)
+
+src_install () {
+	ROOTDIR="${ED}" make gentoo-install
+
+	einstalldocs
+
+	systemd_dounit "${FILESDIR}/${PN}.service"
+}
+
+pkg_postinst() {
+	echo
+	einfo "To make use of greylisting, please update your postfix config."
+	einfo
+	einfo "Put something like this in ${ROOT}/etc/postfix/main.cf:"
+	einfo "    smtpd_recipient_restrictions ="
+	einfo "           ..."
+	einfo "           check_policy_service inet:127.0.0.1:2501"
+	einfo
+	einfo "Remember to restart Postfix after that change. Also remember"
+	einfo "to make the daemon start durig boot:"
+	einfo "  rc-update add sqlgrey default"
+	einfo
+	echo
+	einfo "To setup SQLgrey to run out-of-the-box on your system, run:"
+	einfo "emerge --config ${PN}"
+	echo
+	ewarn "Read the documentation for more info (perldoc sqlgrey) or the"
+	ewarn "included howto ${ROOT}/usr/share/doc/${PF}/HOWTO.gz"
```

Comment:
nitpick, this `.gz` is now a `.bz2`.

Reply:
Changed it to the more general `"included HOWTO in ${ROOT}/usr/share/doc/${PF}/"` as suggested by https://devmanual.gentoo.org/ebuild-writing/common-mistakes/index.html